nrt/data/IEEE30/IEEE30_BasicTestCases/description/IEEE30.tex

```diff
@@ -0,0 +1,206 @@
+
+\begin{tabular}{ | m{2cm} | m{2cm}| m{2cm} | m{2cm} | m{2cm} | }
+  \hline
+  \textbf{Generator} & \textbf{P (MW)} & \textbf{Q (Mvar)} & \textbf{U (kV)} & \textbf{$\Theta$ (°)} \\
+  \hline
+  1 & 260.95 & -16.79 & 139.20 & 0.00 \\
+  \hline
+  2 & 40.00 & 50.00 & 137.94 & -5.35 \\
+  \hline
+  5 & 0.00 & 36.85 & 133.32 & -14.16 \\
+  \hline
+  8 & 0.00 & 37.14 & 133.32 & -11.81 \\
+  \hline
+  11 & 0.00 & 16.17 & 11.90 & -14.11 \\
+  \hline
+  13 & 0.00 & 10.62 & 11.78 & -14.93 \\
+  \hline
+\end{tabular} \\
+
+\section{Scenarios}
+
+The following scenarios are presented based on the DynaSwing, DynaFlow and DynaWaltz documentation : \\
+
+1) Three-phase fault \\
+2) Disconnection of a line \\
+3) Generator disconnection
+
+\chapter{Scenario 1 : Three-phase fault at bus 2}
+
+The simulated scenario is three-phase fault (R = 10e-4 $\Omega$, X = 10e-4 $\Omega$) at bus 2, which corresponds
+to the point of connection for generator 2. The fault lasts from 1 to 1.1 s.
+
+\section{Models}
+
+\begin{itemize}
+  \item Generators : four windings synchronous generators with a proportional voltage regulator and a proportional speed governor. Saturations are considered.
+  \item Frequency model: explicit frequency model OmegaRef using different generator speeds
+  \item Loads: alpha-beta load
+\end{itemize}
+
+\section{Solver}
+
+IDA : order 2, absolute accuracy $10^{-4}$, relative accuracy $10^{-4}$.
+
+\section{Results}
+
+The response at bus 2 is analyzed. The voltage at node 2 falls to zero and consequently the active
+power too. After the fault is cleared, both are stabilized. The machines dynamic data and the
+explicit frequency model parameters should be adapted to achieve the desired response.
+
+\begin{figure}[H]
+  \centering
+    \subfigure[]{\includegraphics[width=0.49\textwidth]{NodeFault - g2 - P.png}}
+    \subfigure[]{\includegraphics[width=0.49\textwidth]{NodeFault - g2 - Q.png}}
+    \subfigure[]{\includegraphics[width=0.49\textwidth]{NodeFault - g2 - omega.png}}
+    \subfigure[]{\includegraphics[width=0.49\textwidth]{NodeFault - bus 2 - voltage.png}}
+    \caption{Generator 2 response to three-phase fault}
+\end{figure}
+
+Generator 1 is electrically close to generator 2. Therefore, when the fault is applied at bus 2, the reactive power injection of generator 1 contributes to limit the voltage dip on bus 1. The active power injection of generator 1 increases to compensate the null injection coming from generator 2 and after the fault clears, oscillates about 8 seconds before returning to its initial value.
+
+\begin{figure}[H]
+  \centering
+    \subfigure[]{\includegraphics[width=0.49\textwidth]{NodeFault - g2 - P.png}}
+    \subfigure[]{\includegraphics[width=0.49\textwidth]{NodeFault - g2 - Q.png}}
+    \subfigure[]{\includegraphics[width=0.49\textwidth]{NodeFault - g2 - omega.png}}
+    \subfigure[]{\includegraphics[width=0.49\textwidth]{NodeFault - bus 2 - voltage.png}}
+    \caption{Generator 1 response to three-phase fault}
```

Comment:
Change the png file, it's g1 and bus 1. Thanks.

Reply:
Corrected